docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,239 @@
+# Kubernetes Basic Windows User Guide
+This guide gives an example on how to use SMB share(s) in pods running on Windows.
+
+## Prerequisites
+- Minimum K8s version: 1.16
+- A non-cluster Windows 1809 VM with `Full` access to **Storage** under `Cloud API Access Scopes`. More [details](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes).
+- An updated personal fork of [kubernetes](https://github.com/kubernetes/kubernetes),[node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar) and [gcp-filestore-csi-driver](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver).
```

Comment:
Are all the patches that you needed to do now merged, or is there more that's still outstanding?

Reply:
These patches aren't meant to be merged as they're just workarounds for some things. The CSI node driver registrar one has some hard coded things to work with CSI v0 and the kubernetes one has some kubelet hacks to get it working.

Reply:
What about the filestore driver patches?

Reply:
Oh right, I think there's a Kubelet workaround implemented in this PR. After this is merged the demo won't work yet because the yaml files need to be edited with project-specific details.

Reply:
Oh you mean the change to delete the mount directory? I think that's fine to merge